ci/build-gdk.ps1

```diff
@@ -1,16 +1,21 @@
+# Expects gdk_home, which is not the GDK location in the engine
 param(
-  [string] $target_platform = "Win64"
+  [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine",
```

Comment:
@KeyMaster- mentioned, and I agree, that this is quite convoluted. The current path to the symlinked engine is C:/b/1/UnrealEngine accessed from C:/b/1/unrealgdk-premerge/../UnrealEngine. Tilman's previous solution was C:/UnrealEngine, but I moved it (reasoning that it is specific to this build and should live in the "build" directory as I understand it). Worth a discussion maybe, no idea how CI works.

Reply:
I've been thinking about it more, and essentially this just means that we place it next to the UnrealGDK checkout folder, which seems sensible. I think with a comment explaining that it will be easier to understand.

Reply:
We share build agents for several different build pipelines, which could be requiring different versions of Unreal Engine, so it might be clearer to put it in the actual checkout directory. Sounds like you guys don't like that though, what is the reason for it?

Reply:
If the engine is linked into the checkout directory, we end up with circular symlinks - i.e. C:\b\UnrealGDK and C:\b\UnrealGDK\UnrealEngine\Engine\Plugins\UnrealGDK would point to the same folder. I didn't run into any issues with that directly, but I'm not sure whether everything will handle that setup correctly.